lib/index.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
 
 var parse = require('./parse')
+var element = require('./element')
+var Element = element.Element
+var DOMElement = require('./dom-element')
 
 /**
  * The only (relevant) data structure
  */
-exports.Element = require('./dom-element')
+exports.Element = DOMElement
```

Comment:
I really don't understand why we did that in the first place, I think ltx.Element should resolve to element and ltx.DOMElement to dom-element